hal/src/common/rtc.rs

```diff
@@ -217,7 +220,7 @@ impl Rtc<Count32Mode> {
 
     /// Returns the internal counter value.
     #[inline]
-    pub fn count32(&mut self) -> u32 {
+    pub fn count32(&self) -> u32 {
```

Comment:
This change is needed because it is otherwise impossible to implement `try_now` in the `Clock` trait from `embedded-time`.